bakery/src/tests/test_bakery_scripts.py

```diff
@@ -857,3 +859,245 @@ def test_gdocify_book(tmp_path, mocker):
         "//x:a[@href]", namespaces={"x": "http://www.w3.org/1999/xhtml"},
     ):
         assert expected_links_by_id[node.attrib["id"]] == node.attrib["href"]
+
+
+def test_copy_resource_s3(tmp_path, mocker):
+    """Test copy_resource_s3 script"""
+    resource_name = "fffe62254ef635871589a848b65db441318171eb"
+
+    test_resource = os.path.join(TEST_DATA_DIR, resource_name)
+
+    book_dir = tmp_path / "col11762"
+    book_dir.mkdir()
+
+    resources_dir = book_dir / "resources"
+    resources_dir.mkdir()
+    resource = resources_dir / "fffe62254ef635871589a848b65db441318171eb.json"
+
+    test_data_one = open(test_resource, "rb").read()
+    resource.write_bytes(test_data_one)
```

Comment:
Perhaps I'm confused, but is this going to write the data from `src/test/data/fffe62254ef635871589a848b65db441318171eb` to `{tmp_path}/col11762/resources/fffe62254ef635871589a848b65db441318171eb.json`?

Reply:
with the same regard to this, 
"ah, there was much confusion here, I started on this test, and got it to pass with one resource but jankily and then moved to the other tests and forgot to come back to make the changes to include checks for two resources here, should be good now. thank you!"

thank you!